addons/project_todo/static/tests/todo_done_checkmark.test.js

```diff
@@ -0,0 +1,186 @@
+import { test, expect, describe } from "@odoo/hoot";
+import { queryFirst, queryAll, queryOne } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { click } from "@mail/../tests/mail_test_helpers";
+import { mountView, onRpc, mountWithCleanup, getService, contains } from "@web/../tests/web_test_helpers";
+import { defineTodoModels } from "./todo_mock_models";
+import { WebClient } from "@web/webclient/webclient";
+
+defineTodoModels();
+describe.current.tags("desktop");
```

Comment:
```suggestion
import { test, expect, describe } from "@odoo/hoot";
import { queryFirst, queryAll, queryOne } from "@odoo/hoot-dom";
import { animationFrame } from "@odoo/hoot-mock";

import { mountView, onRpc, mountWithCleanup, getService, contains } from "@web/../tests/web_test_helpers";
import { WebClient } from "@web/webclient/webclient";
import { click } from "@web/../tests/helpers/utils";

import { click } from "@mail/../tests/mail_test_helpers";

import { defineTodoModels } from "./todo_mock_models";

describe.current.tags("desktop");
defineTodoModels();

```

I believe grouping the imports